src/com/amplitude/api/DeviceInfo.java

```diff
@@ -262,6 +291,8 @@ public boolean isLimitAdTrackingEnabled() {
         return getCachedInfo().limitAdTrackingEnabled;
     }
 
+    public boolean isGPSEnabled() { return getCachedInfo().gpsEnabled; }
```

Comment:
`isGooglePlayServicesEnabled()`

(GPS could be confused with Global Positioning System)